tasks/install-node-exporter.yml

```diff
@@ -105,11 +105,11 @@
 
 - name: copy INIT script to server
   template: src="../templates/node_exporter.sysvinit.{{ ansible_os_family|lower }}.sh.j2"  dest="/etc/init.d/node_exporter"  mode="a+x"
-  when: not prometheus_node_exporter_use_systemd|bool
+  when: not prometheus_use_systemd|bool
 
 - name: copy systemd config to server
   template: src="../templates/node_exporter.service.j2"  dest="/lib/systemd/system/node_exporter.service"
```

Comment:
Now I see why the other configuration files went to `/lib/systemd/system/`. But shouldn't this be `/etc/systemd/system/`...? I'm coming from [here](https://unix.stackexchange.com/a/208352/30457).

Reply:
yes, you are correct